### DISCUSSION TITLE
🚀 Feature: Mobile Ansicht der Tabellenkomponente

### DISCUSSION THREAD

**Discussion by Claymore0307** [public-ui/kolibri#6976] [Ideas]
### Goals

1.Daten bleiben auch in der mobilen Ansicht gut strukturiert und übersichtlich sichtbar.
2.Die Barrierefreiheit der Tabelle wird dabei vollständig beibehalten.

### Background

In unserem Projekt sollen Teile der Anwendung auch in der mobilen Version nutzbar sein. Dabei stellt sich die Herausforderung, wichtige Informationen in den Tabellen weiterhin übersichtlich darzustellen. Aus unserer Sicht reicht seitliches Scrollen allein hierfür nicht aus.

### Proposal

Idealerweise würden die Inhalte in Blöcke umgebrochen werden um soll den Inhalt weiterhin komplett anzeigen zu können.
Hier Beispiele die es bei anderen Komponenten gibt.

1. Primereact DataTable https://codesandbox.io/s/modest-satoshi-nyvt8z
2. react-super-responsive-table https://www.npmjs.com/package/react-super-responsive-table

**Comment by deleonio** [comment]
Vielen Dank @Claymore0307,

ich würde das Thema an unser BITV-Team weitergeben und hierzu nähere Informationen einholen.

VG

**Comment by deleonio** [comment]
@AntnSaj könnt Ihr Euch bitte diese Frage anschauen und uns Feedback geben?

**Comment by deleonio** [comment]
Wir haben uns mal die Samples angeschaut:
- der thead bleibt auch im Mobil
- der Header-Text kommt über einen span im td
- `_variant` ?!
![image](https://github.com/user-attachments/assets/49ab97b9-c165-4cf2-b10a-ff7945809b26)

**Comment by deleonio** [comment]
Wird mit Feature-Ticket https://github.com/public-ui/kolibri/issues/7101 bearbeitet.
